packages/asset-buyer/src/asset_buyer.ts

```diff
@@ -177,6 +179,40 @@ export class AssetBuyer {
         const buyQuote = this.getBuyQuoteAsync(assetData, assetBuyAmount, options);
         return buyQuote;
     }
+    /**
+     * Returns information about available liquidity for an asset
+     * Does not factor in slippage or fees
+     * @param   assetData           The assetData of the desired asset to buy (for more info: https://github.com/0xProject/0x-protocol-specification/blob/master/v2/v2-specification.md).
+     * @param   options             Options for the request. See type definition for more information.
+     *
+     * @return  An object that conforms to LiquidityForAssetData that satisfies the request. See type definition for more information.
+     */
+    public async getLiquidityForAssetDataAsync(
+        assetData: string,
+        options: Partial<LiquidityRequestOpts> = {},
+    ): Promise<LiquidityForAssetData> {
+        const shouldForceOrderRefresh =
+            options.shouldForceOrderRefresh !== undefined ? options.shouldForceOrderRefresh : false;
+        assert.isString('assetData', assetData);
```

Comment:
We're asserting string here vs hex string which we are doing in other places, but actually it looks like `asset_data_utils.ts` has a new method `validateAssetDataOrThrow` which will actually check the shape of the string and make sure it is valid in the context of erc20, erc721, etc, as well as produce a more helpful error message